src/api/v1/db/awsS3/pets-dao-example.js

```diff
@@ -1,9 +1,9 @@
-const capitalize = require('capitalize');
-const _ = require('lodash');
+import capitalize from 'capitalize';
+import _ from 'lodash';
 
-const { serializePets, serializePet } = require('../../serializers/pets-serializer');
+import { serializePets, serializePet } from '../../serializers/pets-serializer';
```

Comment:
As we discussed in person, could we also use something like [alias in webpack](https://webpack.js.org/configuration/resolve#resolvealias) here?

Reply:
Right, we can add aliases using the [babel-plugin-module-resolver](https://github.com/tleunen/babel-plugin-module-resolver/blob/master/DOCS.md#alias) plugin. For this path, it should probably just be changed to `api/v1/serializers/pets-serializer` though.